L/libcgal_julia/build_tarballs.jl

```diff
@@ -85,10 +84,10 @@ products = [
 dependencies = [
     BuildDependency("libjulia_jll"),
     BuildDependency("GMP_jll"),
-    BuildDependency("MPFR_jll"),
+    BuildDependency(VERSION >= v"1.10.0" ? PackageSpec(; name="MPFR_jll", version=v"4.1.0") : "MPFR_jll"),
```

Comment:
This doesn't do what you think it dos: `VERSION` is the version of current Julia session, not the target platform.

Reply:
We had the same idea at the same time! 😁 Actually I had this idea before when "v" was missing....

Reply:
it is what I want the julia version running the wrapper... we'll see if it works or not...

Reply:
@giordano You are right, the dependency is not executed in the wrapper. I don't know if there is somme solution to generate wrapper with different dependencies depending on julia version.

Reply:
@giordano Do you think that we can define and add these lines at the end to generate the tarballs for julia 1.10 qnd 1.11 :
```julia
julia_versions_dev = [v"1.10.0", v"1.11.0",]
julia_compat_dev = join(map(julia_versions_dev) do v "~$(v.major).$(v.minor)" end, ", ")
dependencies_dev = [
    BuildDependency("libjulia_jll"),
    BuildDependency("GMP_jll"),
    BuildDependency(PackageSpec(; name="MPFR_jll", version="4.1.0")),


    Dependency("CGAL_jll", compat="~5.5"),
    Dependency("libcxxwrap_julia_jll", compat="0.9.2"),
]
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies_dev;
    preferred_gcc_version = v"8",
    julia_compat_dev)
```

Reply:
Ok it does not work.....